django/river/common/analyzer/analyzer.py

```diff
@@ -65,40 +86,39 @@ def analyze_mapping(self, resource_mapping):
         return self._cur_analysis
 
     def analyze_filter(self, filter_):
-        col = SqlColumn(
-            filter_["sqlColumn"]["owner"]["name"], filter_["sqlColumn"]["table"], filter_["sqlColumn"]["column"]
-        )
+        col_data = self._columns_data[filter_["sql_column"]]
+        sql_col = SqlColumn(col_data["owner"], col_data["table"], col_data["column"])
 
-        filter_joins = self.parse_joins_mapping(filter_["sqlColumn"]["joins"])
+        filter_joins = self.parse_joins_mapping(col_data["joins"])
         for join in filter_joins:
-            col.add_join(join)
+            sql_col.add_join(join)
 
-        sql_filter = SqlFilter(col, filter_["relation"], filter_["value"])
+        sql_filter = SqlFilter(sql_col, filter_["relation"], filter_["value"])
         self._cur_analysis.add_filter(sql_filter)
 
     def analyze_attribute(self, attribute_mapping):
         logger.debug(
             {
-                "message": f"Analyze attribute {attribute_mapping['path']} {attribute_mapping['definitionId']}",
+                "message": f"Analyze attribute {attribute_mapping['path']} {attribute_mapping['definition_id']}",
                 "resource_id": self._cur_analysis.resource_id,
             },
         )
         # we need to remove the resource_type from the
         # beginning of the path in order to build the fhir object
         # eg: ("Patient.birthDate" --> "birthDate")
         path = re.sub(
-            r"^{resource_type}\.".format(resource_type=self._cur_analysis.definition["type"]),
+            r"^{resource_type}\.".format(resource_type=self._cur_analysis.definition_id),
```

Comment:
can you also link these lines this is the issue I mentioned earlier ?
EDIT: done.